Build failure on LTS 11.3

`proto3-suite` fails to build with the following error:
```
    /Users/jkachmar/src/core-id-tokenmint-experiment/haskell/.stack-work/downloaded/t0CKyCodfmoz/src/Proto3/Suite/DotProto/Generate.hs:332:21: error:
        Ambiguous occurrence ‘liftEither’
        It could refer to either ‘Control.Monad.Except.liftEither’,
                                 imported from ‘Control.Monad.Except’ at src/Proto3/Suite/DotProto/Generate.hs:34:1-37
                                 (and originally defined in ‘Control.Monad.Error.Class’)
                              or ‘Proto3.Suite.DotProto.Generate.liftEither’,
                                 defined at src/Proto3/Suite/DotProto/Generate.hs:80:1
        |
    332 |       do import_ <- liftEither . first CompileParseError =<< importProto searchPaths toplevelFP path
        |                     ^^^^^^^^^^
```
...using the following `stack.yaml` file:
```yaml
resolver: lts-11.3
packages:
  - .
extra-deps:
  - aeson-1.1.2.0
  - QuickCheck-2.9.2
  - swagger2-2.1.6
  - tasty-0.11.3
  - tasty-hunit-0.9.2
  - tasty-quickcheck-0.8.4

  - git: https://github.com/awakesecurity/gRPC-haskell.git
    commit: 3ee4dabc2dfdf7f55b3cc12576de53cd4b767655
    subdirs: 
      - '.'
      - core

  - git: https://github.com/awakesecurity/proto3-suite.git
    commit: 345263a8ec5ff2238d0e9208767a15482bf7270d

  - git: https://github.com/awakesecurity/proto3-wire.git
    commit: c076246ca3d933f2145919f8f6b4809e73a8ab89
```

Looks like this behavior was introduced in this commit (EDIT2: `liftEither` was introduced here, but it didn't become an issue until `mtl-2.2.2` introduced `Control.Monad.liftEither`): c577889f931464dcd95413f58ec18961060910dc

EDIT: Yeah, looks like this builds as expected when I pull in this commit instead: 11fc7cce58e754d1495dabe65dbfe62fb23bb38f

--- a/src/Proto3/Suite/DotProto/Generate.hs
+++ b/src/Proto3/Suite/DotProto/Generate.hs
@@ -1,5 +1,4 @@
--- | This module provides functions to generate Haskell declarations for proto buf messages
-
+{-# LANGUAGE CPP               #-}
 {-# LANGUAGE FlexibleContexts  #-}
 {-# LANGUAGE LambdaCase        #-}
 {-# LANGUAGE MultiWayIf        #-}
@@ -9,6 +8,10 @@
 {-# LANGUAGE RecordWildCards   #-}
 {-# LANGUAGE TupleSections     #-}
 {-# LANGUAGE ViewPatterns      #-}
+
+{-| This module provides functions to generate Haskell declarations for protobuf
+    messages
+-}
 
 module Proto3.Suite.DotProto.Generate
   ( CompileError(..)
@@ -76,11 +79,13 @@ data CompileError
   | Unimplemented           String
     deriving (Show, Eq)
 
+#if !(MIN_VERSION_mtl(2,2,2))
 liftEither :: MonadError e m => Either e a -> m a
 liftEither x =
     case x of
         Left  e -> throwError e
         Right a -> return a
+#endif
 
 -- | Generate a Haskell module corresponding to a @.proto@ file
 compileDotProtoFile

